### DISCUSSION TITLE
The sidebar official configuration is incorrectly written. The correct method is as follows:

### DISCUSSION THREAD

**Discussion by yt46767** [vuepress/core#1035] [Q&A] [answered]
The configuration item sidebar referenced by vuepress(v2.0.0-beta.49) is wrong（https://v2.vuepress.vuejs.org/zh/reference/default-theme/config.html#sidebar）. The correct writing method should be：
```javascript
module.exports = {
  theme: defaultTheme({
    // 侧边栏对象
    // 不同子路径下的页面会使用不同的侧边栏
    sidebar: {
      '/guide/': [
        {
          text: 'Guide',
          children: ['README', 'getting-started'],
        },
      ],
      '/reference/': [
        {
          text: 'Reference',
          children: ['cli', 'config'],
        },
      ],
    },
  }),
}
```
The official usage is that sidebar will not be displayed. In fact, it can be displayed as long as the file name string.

**Comment by Mister-Hope** [comment]
Why don't you try to figure out why these work?????????????????????????????????????????

https://github.com/vuepress/vuepress-next/blob/main/docs/.vuepress/configs/sidebar/en.ts

**Reply by yt46767** [reply]
There should be a bug in it. Can God tell us why?

**Comment by yt46767** [comment, marked as answer]
Also, it should be noted that the folder name is in Chinese, and the sidebar cannot be rendered. The English name is OK

**Reply by yt46767** [reply]
There is a better way: Link address add encodeURI processing. In this way, Chinese URLs can be perfectly supported
![image](https://user-images.githubusercontent.com/10807151/188297783-822111b4-462b-4d1b-8f33-cd6b65f3c045.png)
